frontend/src/views/DashboardPage/DashboardPage.tsx

```diff
@@ -1155,3 +1420,1449 @@ export const DashboardPage = () => {
     </div>
   );
 };
+
+
+
+
+
+
+
+
+
+
+
+
+// import { useCallback, useEffect, useRef, useState } from "react";
```

Comment:
?????

Reply:
This was a commented code, would remove it.